core/src/main/java/org/openmbee/sdvc/core/services/EventService.java

```diff
@@ -0,0 +1,7 @@
+package org.openmbee.sdvc.core.services;
```

Comment:
not sure if we need an event service or multiple impls of event publisher, i think just have a publisher ready in core should be enough..